lib/active_triples/relation.rb

```diff
@@ -233,13 +254,13 @@ def class_for_property
         klass
       end
 
+      ##
+      # @return [RDF::Term] the subject of the relation
       def rdf_subject
-        raise ArgumentError, "wrong number of arguments (#{value_arguments.length} for 1-2)" if value_arguments.length < 1 || value_arguments.length > 2
-        if value_arguments.length > 1
-          value_arguments.first
-        else
-          parent.rdf_subject
-        end
+        raise ArgumentError, 'wrong number of arguments' \
```

Comment:
Is there a reason for this refactor? It feels like it's harder to mentally parse the combination of ternaries, spacing, and postfix conditionals.